policyengine_us/parameters/gov/states/hi/tax/income/deductions/itemized/cap/agi.yaml

```diff
@@ -0,0 +1,29 @@
+description: Hawaii qualifies the filer for the total itemized deductions amount with adjusted gross income of up to this amount, based on filing status.
+SINGLE:
+  2018-01-01: 166_800
+JOINT:
+  2018-01-01: 166_800
+SEPARATE:
+  2018-01-01: 83_400
+HEAD_OF_HOUSEHOLD:
+  2018-01-01: 166_800
+WIDOW:
+  2018-01-01: 166_800
+metadata:
+  breakdown: filing_status
+  unit: currency-USD
+  label: Hawaii itemized deductions adjusted gross income cap
+  reference:
+    - title: 2022 Hawaii Resident Income Tax Instructions Itemized Deductions Worksheet
+      href: https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=32
+    - title: 2021 Hawaii Resident Income Tax Instructions Itemized Deductions Worksheet
+      href: https://files.hawaii.gov/tax/forms/2021/n11ins.pdf#page=32
+    - title: 2020 Hawaii Resident Income Tax Instructions Itemized Deductions Worksheet
+      href: https://files.hawaii.gov/tax/forms/2020/n11ins.pdf#page=34
+    - title: 2019 Hawaii Resident Income Tax Instructions Itemized Deductions Worksheet
+      href: https://files.hawaii.gov/tax/forms/2019/n11ins.pdf#page=38
+    - title: 2018 Hawaii Resident Income Tax Instructions Itemized Deductions Worksheet
+      href: https://files.hawaii.gov/tax/forms/2018/n11ins.pdf#page=38
+    # Legal code does specify the parameter value
+    - title: Hawaii Income Tax Law, Chapter 235, Section 235-2.4, (c)
```

Comment:
does Hawaii reference an IRS program etc? How is the cap defined in the law

Reply:
add reference for all IRS code